tests/cases/compiler/checkIndexConstraintOfJavascriptClassExpression.ts

```diff
@@ -1,10 +1,18 @@
 // @Filename: weird.js
 // @allowJs: true
+// @checkJs: true
+// @strict: true
+// @noEmit: true
 // @out: foo.js
 someFunction(function(BaseClass) {
+        'use strict';
 	class Hello extends BaseClass {
+                const DEFAULT_MESSAGE = "nop!";
```

Comment:
That does clean up the test output a little. #15616 reproed even with it in the incorrect place, though.